ipatests/test_integration/test_caless.py

```diff
@@ -355,7 +355,7 @@ def verify_installation(self):
 
         Called from every positive server install test
         """
-        with open(self.pem_filename) as f:
+        with open(self.pem_filename, 'rb') as f:
             expected_cacrt = f.read()
         logger.debug('Expected /etc/ipa/ca.crt contents:\n%s',
                      expected_cacrt)
```

Comment:
You'll need a decoded `expected_cacrt` here.

Reply:
done